cli/tsc/dts/lib.deno.ns.d.ts

```diff
@@ -3082,37 +3082,37 @@ declare namespace Deno {
     dev: number;
```

Comment:
This one too. I can't do a suggestion on it, but it should be reverted along with its doc comment.